mods/ts/maps/sunstroke/rules.yaml

```diff
@@ -0,0 +1,36 @@
+World:
```

Comment:
Because these are temporary placeholders, and will eventually be removed.  Writing a robust importer would take a disproportionate amount of effort compared to setting the maps up manually.